lizmap/www/js/edition.js

```diff
@@ -776,10 +764,13 @@ OpenLayers.Geometry.pointOnSegment = function(point, segment) {
                     // Get SRID and transform geometry
                     var srid = $('#edition-point-coord-crs').val();
                     vertex.transform( editionLayer['ol'].projection,'EPSG:'+srid );
-                    if ( !$('#edition-point-coord-x').attr('disabled') )
+                    if (srid === "4326"){
+                        $('#edition-point-coord-x').val(vertex.x.toFixed(6));
+                        $('#edition-point-coord-y').val(vertex.y.toFixed(6));
+                    }else{
                         $('#edition-point-coord-x').val(vertex.x);
```

Comment:
```suggestion
                         $('#edition-point-coord-x').val(vertex.x.toFixed(3));
```

Reply:
Are 3 decimals enough for any other cases ?

Reply:
Can we check if the projection is geographic ?